frontend/src/scss/PrimeStyles.scss

```diff
@@ -206,3 +208,18 @@
 .Toastify__toast {
   padding: 3px;
 }
+
+// very basic required-field indicator and validator
+// can't use outline because USDS style already overloads it
+input:required {
+  background-color: rgba(255, 247, 237, 0.85);
+}
+
+input:valid {
```

Comment:
I don't think we'd be using HTML5 validation since it's not very flexible. Whenever validation is added we'd probably use text messages along with the error highlighting.

Reply:
Agreed. I was using it as a _something_ until we get a better system in place for the whole project. Should I remove it? Should I add a `todo`?